src/os/ui/featureedit.js

```diff
@@ -980,8 +1010,20 @@ os.ui.FeatureEditCtrl.prototype.loadFromFeature_ = function(feature) {
       if (coordinate) {
         this['pointGeometry'] = {
           'lon': coordinate[0],
-          'lat': coordinate[1]
+          'lat': coordinate[1],
+          'alt': coordinate[2]
         };
+
+        var altU = this['options']['feature']['values_']['ALTITUDE_UNITS'];
```

Comment:
`this['options']['feature']` is the same object as the `feature` param to this function. Use that instead, and retrieve the value of `altU` with `feature.get(os.Fields.ALT_UNITS)`.